pygenn/genn_model.py

```diff
@@ -969,7 +969,7 @@ def create_custom_init_var_snippet_class(class_name, param_names=None,
 
     body = {}
 
-    if get_code is not None:
```

Comment:
All the problem was is that ``get_code`` here should have been ``var_init_code`` - the parameter that got passed in!